src/main/java/training/subsystems/Roller/Roller.java

```diff
@@ -0,0 +1,73 @@
+package training.subsystems.Roller;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import utils.GBSubsystem;
+
+import static training.subsystems.Roller.RollerConstants.*;
+import static training.subsystems.Roller.RollerDirection.kBackward;
+import static training.subsystems.Roller.RollerDirection.kForward;
+
+public class Roller extends GBSubsystem {
+    private static Roller instance;
+
+    public static Roller getInstance() {
+        if (instance == null) {
+            instance = new Roller();
+        }
+        return instance;
+    }
+
+    private final CANSparkMax motor;
+    private final SparkPIDController motorPID;
```

Comment:
```suggestion
    private final SparkPIDController motorPIDController;
```
but dont save it separately, u dont know what happens inside

Reply:
wdym by don't save it separately? like use .getPIDController each time?